src/sql/workbench/parts/dashboard/widgets/properties/media/propertiesWidget.css

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.flex-table-row {
+	display: flex;
```

Comment:
The selectors in this file all need to be more fully qualified - you should be able to use the angular component selector for this like src\sql\workbench\parts\notebook\cellViews\textCell.css does.